scripts/load_secrets.sh

```diff
@@ -0,0 +1,12 @@
+if [ ! -d "../official-models" ]; then
```

Comment:
😍

Reply:
I'm actually not sure if this is ideal to have committed here tbh

Reply:
Good point, I'm also not sure.

Given this is a public repo, perhaps it'd be better to migrate this into the `official-models` repo, and then you'd run from there to push all of the `.env`s to the right place?